File: Java/Program/Shirt.java
public class Shirt extends Clothing {
    String color, sleeve_type;

    public Shirt(){
    }

    public Shirt(String color, String sleeve_type){
        this.color = color;
        this.sleeve_type = sleeve_type;
    }
    
    public String getColor(){
        return this.color;
    }

    public void setColor(String color){
        this.color = color;
    }

    public String getSleeve_type(){
        return this.sleeve_type;
    }

    public void setSleeve_type(String sleeve_type){
        this.sleeve_type = sleeve_type;
    }
}

File: PHP/Program/Shirt.php
<?php
    require('Clothing.php');

    class Shirt extends Clothing{
        private $color = '';
        private $sleeve_type = '';

        public function __construct($idProduct = '', $name = '', $brand = '', $price, $size = '', $material = '', $gender = '', $color = '', $sleeve_type = ''){
            parent::__construct($idProduct, $name, $brand, $price, $size, $material, $gender);
            $this->color = $color;
            $this->sleeve_type = $sleeve_type;
        }

        public function getColor(){
            return $this->color;
        }

        public function setColor($color){
            $this->color = $color;
        }

        public function getSleeve(){
            return $this->sleeve_type;
        }

        public function setSleeve($sleeve_type){
            $this->sleeve_type = $sleeve_type;
        }
    }
?>